### ISSUE TITLE
scrollParent constraints not respected

### ISSUE THREAD

**Issue by davewasmer** [pzuraq/liquid-tether#60]
It appears that any constraints that target `'scrollParent'` are not respected. My best guess is that this is because the wormhole warps the element outside the expected scrollParent, so Tether.js has sees the window itself as the scrollParent.

I'm not sure there's an easy solution here. Would it be possible to supply your own liquid tether wormhole target? If that was the case, you could just inject the wormhole target at whatever level of the DOM you want to achieve the appropriate scrollParent.

At the very least, the docs should probably be updated to reflect the lack of support for this aspect of Tether.js

**Comment by pzuraq** [comment]
Yeah, that makes sense. `liquid-wormhole` warps everything to the top level of the DOM, and then `liquid-tether` applies the tether after the warp. 

It may take a bit more work than just being able to place a `liquid-target` manually. I actually forked Tether.js to make it so we could manage tethered element positioning directly. Normally, tether's will position themselves relative to the window, but in our case we wanted them to position themselves relative to the animation container (which liquid-fire is going to be moving around all over the place). I tried to get it to work with Tether.js's standard behavior, but there were too many edge cases regarding styling, etc. to make the two play nicely together.

I'll see what I can do, this will also address liquid-wormhole/#12
